conf/modules.config

```diff
@@ -475,11 +467,11 @@ process {
     withName: '.*ANNOTATE_SNV:ENSEMBLVEP' {
         ext.prefix = { "${vcf.simpleName}_rohann_vcfanno_filter_vep" }
```

Comment:
```suggestion
        ext.prefix = { "${vcf.simpleName}_vep" }
```
We are not running rhocall and vcfanno in this pipeline

Reply:
That's true, fixed